src/lib/__tests__/extendExpect.ts

```diff
@@ -0,0 +1,20 @@
+import { toMatchDOMNodes } from "./matchers";
+import { expect } from "vitest";
+
+interface CustomMatchers<R = unknown> {
+  toMatchDOMNodes(expected: HTMLElement | Node[] | string): R;
+}
+
+declare global {
+  /* eslint-disable-next-line @typescript-eslint/no-namespace */
+  namespace Vi {
```

Comment:
This will need to be updated if we merge the vitest dependency update (#105) first.